doc/source/serve/doc_code/deployment_graph_combine_two_nodes_with_passing_input_parallel.py

```diff
@@ -0,0 +1,32 @@
+import ray
+from ray import serve
+from ray.experimental.dag.input_node import InputNode
+
+
+ray.init()
+serve.start()
+
+
+@serve.deployment
+class Model:
+    def __init__(self, weight):
+        self.weight = weight
+
+    def forward(self, input):
+        return input + self.weight
+
+
+@serve.deployment
+def combine(value_refs):
+    return sum(ray.get(value_refs))
+
+
+number_nodes = 2
```

Comment:
if it's only two nodes i think the most intuitive way should be something like 

```
    with InputNode() as dag_input:
        output_1 = func_1.bind(dag_input)
        output_2 = func_2.bind(dag_input)
        dag = combine.bind(output_1, output_2)
```

rather than a loop. Do you want to split those as two separate parts, or a loop as natural extension of using plain variable name?

Reply:
make sense for having two nodes instead of loop